create-release.py

```diff
@@ -122,12 +122,12 @@ def update_version_to_release() -> None:
         # Update docker related tags
         sed(_source("Makefile"), r"^TAG:=dev", f"TAG:={new_version}")
         sed(_source("README.md"), r"elyra:dev ", f"elyra:{new_version} ")
-        if config.rc is None and config.beta is None:
-            # Update the stable version Binder link
-            sed(_source("README.md"), r"/v[0-9].[0-9].[0-9]?", f"/v{new_version}?")
+        # if config.rc is None and config.beta is None:
+        # Update the stable version Binder link
+        #    sed(_source("README.md"), r"/v[0-9]\+\.[0-9]\+\.[0-9]\+", f"/v{new_version}?")
```

Comment:
If we're going to comment out the code, we should comment out correct code since it will be assumed that it worked previously...
```suggestion
        #    sed(_source("README.md"), r"/v[0-9]\+\.[0-9]\+\.[0-9]\+?", f"/v{new_version}?")
```

Reply:
removed per initial comment